prov/verbs/src/verbs_info.c

```diff
@@ -361,7 +361,8 @@ int fi_ibv_check_rx_attr(const struct fi_rx_attr *attr,
 		return -FI_ENODATA;
 	}
 
-	if (attr->total_buffered_recv > info->rx_attr->total_buffered_recv) {
+	if (!attr->total_buffered_recv &&
+		(attr->total_buffered_recv > info->rx_attr->total_buffered_recv)) {
```

Comment:
From man page: 

> Defines the total available space allocated by the provider to buffer messages that are received for which there is no matching receive operation. If set to 0, and the domain does not support FI_RM_ENABLED, any messages that arrive before a receive buffer has been posted are lost. When the domain supports FI_RM_ENABLED, the actual amount of buffering provided may exceed the value specified in total_buffered_recv.

The verbs/RDM is able to receive such many unexpected (without preposting) messages as system memory allows. As I understand, this means the provider does not support FI_RM_ENABLED and attr->total_buffered_recv should be set = 0, otherwise we need do second check for EP_MSG